vumi/components/tests/test_message_store.py

```diff
@@ -1249,6 +1250,12 @@ def test_reconcile_cache(self):
         self.assertFalse((yield self.store.needs_reconciliation(batch_id)))
         self.assertFalse(
             (yield self.store.needs_reconciliation(batch_id, delta=0)))
+
+        inbound_uniques = yield self.store.cache.count_from_addrs(batch_id)
+        self.assertEqual(inbound_uniques, 1)
+        outbound_uniques = yield self.store.cache.count_to_addrs(batch_id)
+        self.assertEqual(outbound_uniques, 1)
```

Comment:
Probably because I didn't think about it when I was writing the tests. (The new message store tests do it that way, though.)